.github/workflows/release.yml

```diff
@@ -503,11 +496,10 @@ jobs:
       run: tar -zxf ${{github.workspace}}/sdk/java.tar.gz -C
         ${{github.workspace}}/sdk/java
     - name: Set PACKAGE_VERSION to Env
-      run:
-          echo "PACKAGE_VERSION=$(pulumictl get version --language generic)" >>
+      run: echo "PACKAGE_VERSION=$(pulumictl get version --language generic)" >>
         $GITHUB_ENV
     - name: Publish Java SDK
-      uses: gradle/gradle-build-action@v2
+      uses: gradle/gradle-build-action@9b814496b50909128c6a52622b416c5ffa04db49
```

Comment:
I checked this SHA; it's the 2.70 tag of gradle/gradle-build-action so probably fine to use. 

TIL: security experts recommend using the full SHA rather then a tag b/c it's possible for someone to compromise the action and change the tag to point to different code. Would be nice if we had comments somehow to identify the tag we're using though!